lang/en.json

```
@@ -651,21 +651,34 @@
 "DND5E.PlaceTemplate": "Place Measured Template",
 "DND5E.Polymorph": "Polymorph",
 "DND5E.PolymorphAcceptSettings": "Custom Settings",
+"DND5E.PolymorphCustomTransformation": "Custom Transformation",
+"DND5E.PolymorphEffectTransformation": "Effect Transformation",
 "DND5E.PolymorphKeepBio": "Keep biography",
 "DND5E.PolymorphKeepClass": "Keep proficiency bonus (leaves class items in sheet)",
 "DND5E.PolymorphKeepFeats": "Keep features",
 "DND5E.PolymorphKeepItems": "Keep equipment",
 "DND5E.PolymorphKeepMental": "Keep mental ability scores (Int, Wis, Cha)",
 "DND5E.PolymorphKeepPhysical": "Keep physical ability scores (Str, Dex, Con)",
 "DND5E.PolymorphKeepSaves": "Keep saving throw proficiencies",
+"DND5E.PolymorphKeepSelf": "Keep self (just change image and size i.e. for spells like Disguise Self)",
 "DND5E.PolymorphKeepSkills": "Keep skill proficiencies",
 "DND5E.PolymorphKeepSpells": "Keep spells",
 "DND5E.PolymorphKeepVision": "Keep vision (character and token)",
 "DND5E.PolymorphMergeSaves": "Merge saving throw proficiencies (take the highest)",
 "DND5E.PolymorphMergeSkills": "Merge skill proficiencies (take the highest)",
 "DND5E.PolymorphPromptTitle": "Transforming Actor",
+"DND5E.PolymorphRemoveAE": "Remove All Active effects (overrides other effect retention options).",
+"DND5E.PolymorphKeepBackgroundAE": "Keep effects that originate from your background?",
+"DND5E.PolymorphKeepClassAE": "Keep effects that originate from your class or subclass?",
+"DND5E.PolymorphKeepEquipmentAE": "Keep effects that originate from your equipment?",
+"DND5E.PolymorphKeepFeatureAE": "Keep effects that originate from your features?",
+"DND5E.PolymorphKeepOriginAE": "Keep effects that originate on the actor i.e. effects not attached to features/items/spells",
+"DND5E.PolymorphKeepOtherOriginAE": "Keep effects that originate on a different actor i.e. spells/items/features from opponents",
+"DND5E.PolymorphKeepSpellAE": "Keep effects that originate from your spells?",
```

Looking through the (user-visible) terms used by the rest of the system, I think "effect" (rather than "Active Effect" or "active effect") should be used.

```suggestion
"DND5E.PolymorphRemoveAE": "Remove all effects (overrides other effect retention options).",
"DND5E.PolymorphKeepBackgroundAE": "Keep effects that originate from your background?",
"DND5E.PolymorphKeepClassAE": "Keep effects that originate from your class or subclass?",
"DND5E.PolymorphKeepEquipmentAE": "Keep effects that originate from your equipment?",
"DND5E.PolymorphKeepFeatureAE": "Keep effects that originate from your features?",
"DND5E.PolymorphKeepOriginAE": "Keep effects that originate from this actor (i.e. effects not attached to features, items, or spells)?",
"DND5E.PolymorphKeepOtherOriginAE": "Keep effects that originate from a different actor (i.e. effects attached to another actor's feature, items, or spells)?",
"DND5E.PolymorphKeepSpellAE": "Keep effects that originate from your spells?",
```

---

Ah, thanks! I thought I had changed all of these!